src/pages/Intro.js

```diff
@@ -21,17 +21,28 @@ function Intro({ close, data }) {
+                <Primitives.Paragraphs.Lead>Ønsker du å dele data, men er usikker på om du har lov til å dele data? Vår Veileder hjelper deg med å finne ut om du lovlig kan dele data, eller hva du eventuelt må endre for å kunne dele.</Primitives.Paragraphs.Lead>
+                <Primitives.Heading.H2>Hva inneholder veilederen</Primitives.Heading.H2>
                 <Primitives.Paragraphs.P>
-                Svarene kan du ta vare på slik at du kan huske hva du må finne ut av og kan dokumentere vurderingene du har foretatt. Hvis du har behov for å finne ut av ting kan du komme tilbake for å fortsette veilederen senere.
+                Veilederen guider deg gjennom noen sentrale juridiske spørsmål du må besvare før du deler data.
+                </Primitives.Paragraphs.P>
+                <Primitives.Paragraphs.P>
+                Målgruppen for denne Veilederen er både jurister og ikke-jurister som ønsker svar på spørsmålet “kan jeg dele disse dataene?”. Veiledningen tilpasser seg svarene som oppgis og vurderingene som foretas. I enkelte tilfeller gis det også forklaringer og eksempler som hjelper deg videre om du har behov for mer informasjon.
+                </Primitives.Paragraphs.P>
+                <Primitives.Paragraphs.P>
+                Veiledningen er <u><b>ikke</b></u> uttømmende. Det betyr at den kun hjelper deg med å svare på noen sentrale juridiske spørsmål. Det kan være andre spørsmål som ikke besvares her. Du er selv ansvarlig for å sørge for at din deling av data skjer i henhold til gjeldende regelverk.
+                </Primitives.Paragraphs.P>
+                <Primitives.Heading.H2>Start nå og fullfør senere</Primitives.Heading.H2>
+                <Primitives.Paragraphs.P>
+                Når du er ferdig med Veilederen, kan du laste ned en PDF med resultatene dine. Resultatene gir deg en oversikt over hva du må finne ut av og kan blant annet benyttes som grunnlag for å dokumentere vurderingene som er foretatt. Det er mulig å ta en pause i veilederen ved behov, og komme tilbake for å fortsette senere.
                 </Primitives.Paragraphs.P>
                 <Primitives.Heading.H2>Før du begynner</Primitives.Heading.H2>
                 <Primitives.Paragraphs.P>
-                Før du begynner er det lurt å finne fram til dokumentasjon som inneholder informasjon om dataene du ønsker å dele. Dette er typisk:
+                Det er lurt å finne fram til dokumentasjon som inneholder informasjon om dataene du ønsker å dele. Dette er typisk:
                 </Primitives.Paragraphs.P>
                 <ul>
-                    <li>Internkontroll for virksomhetens behandling av personopplysninger</li>
-                    <li>Behandlingsprotokoll</li>
+                    <li><a href="https://www.datatilsynet.no/rettigheter-og-plikter/virksomhetenes-plikter/informasjonssikkerhet-internkontroll/etablere-internkontroll/">Internkontroll</a> for virksomhetens behandling av personopplysninger</li>
```

Comment:
Hvis lenken skal åpnes i ny fane så må du legge inn target="_blank"
```suggestion
                    <li><a href="https://www.datatilsynet.no/rettigheter-og-plikter/virksomhetenes-plikter/informasjonssikkerhet-internkontroll/etablere-internkontroll/" target=”_blank” rel=”noopener noreferrer”>Internkontroll</a> for virksomhetens behandling av personopplysninger</li>
```

Reply:
La til target="_blank" på alle hyperlenker